.travis.yml

```diff
@@ -15,13 +15,13 @@ matrix:
       install:
         # CHANGED_FILES detects whether or not files related to the nightly build are changed.
         # True is so that the command returns a 0 exit code whether or not CHANGED_FILES is empty.
-        - CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples") || true;
+        - CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples|Dockerfile") || true;
```

Comment:
You'll need to add a backslash before the alternation operator, e.g. `\|Dockerfile` instead of `|Dockerfile`. Right now, the `grep` command looks for changes in files with `tests/examples` and `examples|Dockerfile` patterns, rather than `tests/examples`, `examples`, and `Dockerfile` patterns (which is why the nightly stage isn't activating).

Reply:
I added escape character to the pattern.